tests/conftest.py

```diff
@@ -116,3 +118,49 @@ def init_sqldb(dsn):
     engine = create_engine(dsn)
     Base.metadata.create_all(bind=engine)
     return sessionmaker(bind=engine)
+
+
+class DictToObjectWrapper:
```

Comment:
https://github.com/mewwts/addict or
https://github.com/Infinidat/munch - an actively maintained munch fork by the guys at https://github.com/Infinidat) or
https://github.com/ramazanpolat/prodict - this is a smaller one, but the author promises auto-completion in IDEs which sounds fantastic. I don't know if that works with the others, but if not, then this is a big advantage

I suggest you read this SO answer if you haven't come across it already (https://stackoverflow.com/a/59379520/1875031), or the entire thread if you're bored :)

Reply:
I've read it, and took my solution from some other thread that is referenced somewhere in this thread (can't find it currently)
Generally, all of these packages gives the option to have dict which can be **also** access with object style (`obj.key`), **but** the dict functionalities remain (`.items()`, `.keys()` etc...) + dict style access is still possible (`obj['key']`)
I don't need these, therefore my solution can be much more simpler, all I need is a way to instantiate a mock object from a dict. 
I don't need to change anything on the dict afterwards, I don't need them to be synced, I don't need the object to keep act like a dict, I actually do no want it, e.g. if someone is doing `obj.keys()` is should raise an exception.
Specifically - prodict gives auto-completion if you define (meaning you know it) the schema from ahead, this is not our case, if it was, I could simply use the already in use Pydantic

Reply:
I would strongly suggest to go with one of these (that fits your needs) even if you just need a subset of the functionality instead of re-inventing the wheel (there's a huge selection of premium wheels - pick one that suits you) - but, 🤷  as you wish

Reply:
I don't want to use them not because of the extra functionality I don't need, but because the extra functionality I don't want, e.g. if somewhere in the code the runtime variable is an object, and someone mistakenly do `obj['key']` I want it to fail, If I'll use any of these packages it will succeed